opal/mca/common/ofi/common_ofi.c

```diff
@@ -357,17 +378,18 @@ opal_mca_common_ofi_select_provider(struct fi_info *provider_list, int local_ind
         provider = provider_table[local_index % num_provider];
     }
 
-#if OPAL_DEBUG_ENABLE
+#if OPAL_OFI_PCI_DATA_AVAILABLE
     if (NULL != provider->nic) {
         pci = provider->nic->bus_attr->attr.pci;
         cpusets_match = compare_cpusets(opal_hwloc_topology, pci);
     }
+#endif
 
+#if OPAL_DEBUG_ENABLE
```

Comment:
Sorry, I didn't notice these `opal_output()`s the first time around.  There should be on the `opal_common_ofi.output` stream -- not on fixed number streams like 1 or 10.

There's two `opal_output()`s earlier in this module that seem to be emitting error messages (vs. helpful / info messages) -- should they be converted to `opal_show_help()`?

Reply:
I don't think that needs to go into this patch though. I think we should get this merged so we can get your build running and then we can fix the output issues. I'd say that this is a critical blocker so we should get it merged ASAP.

Reply:
Can you please make another commit / PR / whatever ASAP to fix these fixed-stream `opal_output()`s?  They're actually wrong -- you're writing to streams without having any idea what those streams are.  I.e., those outputs could show up for unexpected reasons.

Reply:
yes I pushed some of the other changes, but I'm working on addressing that as well. I think it'll need internal CR since I'll have to add a help-common-ofi.txt file and we'll have to agree on the error messages/info

Reply:
If you want to take a little more time to fully convert to `opal_show_help()`, no problem.  But you could change the `opal_output(x, ...)` to `opal_output_verbose(...)` in the immediate future or somesuch.

I.e., I agree that converting to `opal_show_help()` is secondary: my primary concern was the hard-coded stream IDs.

Reply:
ok I switched them to `opal_output_verbose(...)` for now, I'll work on adding `opal_show_help()`

Reply:
Thanks!